recipes/densmap-learn/LICENSE

```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
```

Comment:
Where did this license file come from? It looks like a template (see the "<copyright holders>") and I can't find it anywhere in the original repository. I see the original repo says "Licensed under MIT" but that doesn't mean we can just add a license file on our own.

Reply:
In these cases I tend to make a PR upstream to add the licence file and guess at the copyright notice (`Hoon Cho` in this case). If it's accepted then it's fine to continue here.

Reply:
Copied verbatim from https://mit-license.org/. The original repo doesn't contain a license file but the linter etc seem to require one

Reply:
Yeah will do

Reply:
So should I remove this copy of LICENSE after the PR in the main repo is accepted?

Reply:
Yes, you should remove the manually packaged license file here once the upstream repo has it and it's in the tarball used to build the package here

Reply:
Was waiting for the new version to be up on pypi which it is now. Should be good to go unless you spot any issues